docs/content/architecture/pgadmin4.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "pgAdmin 4"
+date:
+draft: false
+weight: 900
+---
+
+![pgAdmin 4 Query](/images/pgadmin4-query.png)
+
+[pgAdmin 4](https://www.pgadmin.org/) is a popular graphical user interface that
+makes it easy to work with PostgreSQL databases from both a desktop or web-based
+client. With its ability to manage and orchestrate changes for PostgreSQL users,
+the PostgreSQL Operator is a natural partner to keep a pgAdmin 4 environment
+synchronized with a PostgreSQL environment.
+
+The PostgreSQL Operator lets you deploy a pgAdmin 4 environment alongside a
+PostgreSQL cluster and keeps users' database credentials synchronized. You can
+simply log into pgAdmin 4 with your PostgreSQL username and password and
+immediately have access to your databases.
+
+## Deploying pgAdmin 4
+
+Let's use the PostgreSQL cluster we created in the [quickstart]({{< relref "quickstart/_index.md" >}})
+as an example. If you don't have the `hippo` example cluster running, follow those steps.
+
+After the PostgreSQL cluster becomes ready, you can create a pgAdmin 4 deployment. To add the
```

Comment:
Just to double-check, what happens if we try to add the `userInterface` field the first time we create a postgres cluster?

Reply:
It does come up as expected. 

I could also update the docs to have it be one step :thinking: . Should I provide one spec with the UI fields already added :question: I was following steps from the old docs

Reply:
I think it makes sense to continue this as if from the quickstart (with everything that implies, e.g., a hippo user), but I'd maybe reword this section a bit in that case, something like "if you've done the quickstart, add this section to the spec and reapply; if you don't have any postgres cluster running, add this section to a spec with a user specified, and apply." (But, you know, better said.)

I just think if I came to this cold and read "After the PostgreSQL cluster becomes ready, you can create a pgAdmin 4 deployment", I might think it was a two-step process.